### PR TITLE
Updatess

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: data/features.json [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+      "Documented?": "",
+      "Feature": "Import a design from GitHub, GitLab, BitBucket",
+      "Function": "Premium Design Import Features",
+      "Pricing page?": "X",
+      "Subscription Tier": "Enterprise",
+      "Tech": "Meshery"
+    }
+  },
+  {
+    "documentation": "https://deploy-preview-379--bejewelled-pegasus-b0ce81.netlify.app/kanvas/designer/export-designs/#exporting-as-a-design-file",
```
Comment (vishalvivekm) [inline data/features.json:87]:
```suggestion
    "documentation": "https://deploy-preview-394--bejewelled-pegasus-b0ce81.netlify.app/kanvas/designer/export-designs/#exporting-as-a-design-file",
```

File: data/features.json [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+      "Feature": "Import, Unpublish, Publish, Download, Edit, Clone, Delete, Details of WASM Filter",
+      "Function": "WASM Filter and filter",
+      "Pricing page?": "X",
+      "Subscription Tier": "Free",
+      "Tech": "Meshery"
+    }
+  },
+  {
+    "pricing_page": "true",
+    "documentation": "https://deploy-preview-379--bejewelled-pegasus-b0ce81.netlify.app/kanvas/designer/comments/",
```
Comment (vishalvivekm) [inline data/features.json:257]:
```suggestion
    "documentation": "https://deploy-preview-394--bejewelled-pegasus-b0ce81.netlify.app//kanvas/designer/comments/",
```

File: data/features.json
Comment (vishalvivekm) [inline data/features.json:1]: this file is included in the commit for the sake of preview

File: .github/workflows/pricing-list.yml
Comment (leecalcote) [inline .github/workflows/pricing-list.yml:1]: Rename this file, @SAHU-01

File: .github/workflows/pricing-list.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      spreadsheet_uri:
+        description: 'Link of the spreadsheet containing subscription details.'
+        type: string
+        required: true
+
+jobs:
+  update-pricing-and-frontmatter:
+    runs-on: ubuntu-latest
+    env:  
+      PRICING_FILE: 'pricing-data.json'
```
Comment (leecalcote) [inline .github/workflows/pricing-list.yml:15]: Rename variable and file, @SAHU-01

File: .github/workflows/pricing-list.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        required: true
+
+jobs:
+  update-pricing-and-frontmatter:
+    runs-on: ubuntu-latest
+    env:  
+      PRICING_FILE: 'pricing-data.json'  
+
+    steps:
+      - name: Trigger Pricing List workflow and wait for completion
```
Comment (leecalcote) [inline .github/workflows/pricing-list.yml:18]: Remove "pricing"

File: .github/workflows/pricing-list.yml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        id: trigger-workflow
+        run: |
```
Comment (leecalcote) [inline .github/workflows/pricing-list.yml:20]: Use GitHub native call, not an API call, @SAHU-01

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: .github/workflows/pricing-list.yml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+          WORKFLOW_ID=$(curl -X POST \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: token ${{ secrets.GH_ACCESS_TOKEN }}" \
+            https://api.github.com/repos/layer5labs/meshery-extensions-packages/actions/workflows/generate-pricing-list.yml/dispatches \
+            -d '{"ref":"master", "inputs":{"spreadsheet_uri":"${{ inputs.spreadsheet_uri }}"}}' \
+            -s -o /dev/null -w "%{http_code}")
+          
+          echo "Triggered workflow, waiting for completion..."
+          
+          while true; do
```
Comment (leecalcote) [inline .github/workflows/pricing-list.yml:30]: What? No.

File: .github/workflows/pricing-list.yml [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+          go-version: '1.21' 
+
+      - name: Run update_frontmatter.go
+        run: go run update_frontmatter.go "${{ env.PRICING_FILE }}" 
+
+      - name: Commit and push if there are changes
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add ${{ env.PRICING_FILE }}
```
Comment (leecalcote) [inline .github/workflows/pricing-list.yml:56]: Rename

File: .github/workflows/pricing-list.yml [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.21' 
+
+      - name: Run update_frontmatter.go
```
Comment (leecalcote) [inline .github/workflows/pricing-list.yml:49]: Is a go program really necessary?

File: .github/workflows/pricing-list.yml [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+        run: go run update_frontmatter.go "${{ env.PRICING_FILE }}" 
+
+      - name: Commit and push if there are changes
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add ${{ env.PRICING_FILE }}
+          git diff --quiet && git diff --staged --quiet || (git commit -m "Update front matter based on new pricing data" && git push)
```
Comment (leecalcote) [inline .github/workflows/pricing-list.yml:57]: It's not pricing

Comment (leecalcote) [inline .github/workflows/pricing-list.yml:57]: Don't use git commands. Use the same autocommitter that we use in many other workflows.

File: assets/scss/_styles_project.scss [hunk 10]
```diff
@@ -381,6 +368,50 @@ a:not([href]):not([class]):hover {
   border-color: #00b39f;
 }
 
+// front-matter
+.matterinfo {
+  font-weight: $font-weight-medium;
+  background: $black;
+  font-family: "Open Sans";
```
Comment (leecalcote) [inline assets/scss/_styles_project.scss:375]: Not Qanelas Soft, huh?

File: data/features.json
Comment (leecalcote) [reply data/features.json:1]: Will this file be persisted to the repo?

File: static/images/subscription.svg
Comment (leecalcote) [inline static/images/subscription.svg:1]: Is this copied from Cloud?

**Pull request closed without merging**